webpack/webpack.release.js

```diff
@@ -15,7 +16,7 @@ module.exports = merge(common, {
   output: {
     path: path.resolve(__dirname, "../dist"),
     publicPath: "/",
-    filename: "[name].bundle.js"
+    filename: "[name].[hash].bundle.js"
```

Comment:
Great! This should avoid the issue of people sometimes having to do Ctrl/Shift refreshes after frontend upgrades

Reply:
Thanks!